### PR TITLE
ES6 modules

### LINKED ISSUE #93: ES6 Module export
Now that you kindly implemented https://github.com/sinonjs/sinon/issues/1805 , may I also request a module exposed for `sinon-test` (including `module` on `package.json`)?

It looks like all you will need is an addition to the Rollup config for the "es" format.

While I could make a PR, besides figuring you may have your own ideas on any tests for it, I was also hoping you might first be able to make an npm bump to `sinon` so the dev dependency here (and for other users of versioned sinon) could get the benefit of exclusively using modules.

Thanks so much!

### PR DESCRIPTION
**Pull request by brettz9**
Builds on PR #94 to fix #93

### CODE REVIEW

**Comment by mroderick** [comment]
I have run the build task locally, and the output looks good.

Before we merge this, someone should try creating a couple of projects that show that consuming the artefacts work. This could be as simple as making a single repo that uses a linked version of `sinon-test` and load the two different builds into a test.

We can use CommonJS for the UMD version, and use [esm](https://github.com/standard-things/esm#readme) for ES module.

**Comment by brettz9** [comment]
FWIW, I'm using sinon-test at https://github.com/SVG-Edit/svgedit/blob/a3f0b8e50196a55780271ecdab6e1c8a0938783a/test/draw_test.js#L7

File: rollup.config.js [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+            format: cfg.format
+        },
+        plugins: [
+            commonjs({sourceMap: false})
+        ]
+    };
+}
+
+module.exports = [
+    setConfig({format: "umd", file: "dist/sinon-test.js"}),
```
Comment (fatso83) [inline rollup.config.js]: Mind renaming the `setConfig` to `createConfig`? `set*` functions usually have side effects, while this is a pure function.

Comment (fatso83) [reply rollup.config.js]: Otherwise, this is a fine improvement.

Comment (brettz9) [reply rollup.config.js]: Thanks, good point... Fixed...

**Comment by fatso83** [comment]
@mroderick I added the repo tests in a new branch that builds upon this. Merging this, then that.

**Comment by fatso83** [comment]
Thanks for your contribution, @brettz9!

**Comment by mroderick** [comment]
Good job @brettz9!

**Pull request merged**